composer/workflows/dataflowtemplateoperator_create_dataset_and_table_test.py

```diff
@@ -0,0 +1,48 @@
+from google.cloud import bigquery
+import google.cloud.exceptions
+import pytest
+from . import dataflowtemplateoperator_create_dataset_and_table_helper as helper
+
+client = bigquery.Client()
+
+expected_schema = schema = [
+    bigquery.SchemaField("location", "GEOGRAPHY", mode="REQUIRED"),
+    bigquery.SchemaField("average_temperature", "INTEGER", mode="REQUIRED"),
+    bigquery.SchemaField("month", "STRING", mode="REQUIRED"),
+    bigquery.SchemaField("inches_of_rain", "NUMERIC", mode="NULLABLE"),
+    bigquery.SchemaField("is_current", "BOOLEAN", mode="NULLABLE"),
+    bigquery.SchemaField("latest_measurement", "DATE", mode="NULLABLE"),
+]
+
+expected_table_id = f"{client.project}.average_weather.average_weather"
+expected_dataset_id = f"{client.project}.average_weather"
+
+
+@pytest.fixture(scope="module")
+def dataset():
+    try:
+        dataset = client.get_dataset(expected_dataset_id)
```

Comment:
This check has a race condition. Consider when.two CI builds are simultaneously run the same script.

Instead, we should use unique id as in the comment above. If we use unique dataset id, we don't have to check the existence.

Also, I have more comments on this fixture. Please see a comment in the test body below.